pypureclient/flasharray/FA_2_0/client.py

```diff
@@ -20,7 +20,7 @@ class Client(object):
     DEFAULT_TIMEOUT = 15.0
     DEFAULT_RETRIES = 5
     # Format: client/client_version/endpoint/endpoint_version/system/release
-    USER_AGENT = ('pypureclient/1.24.0/FA/2.0/{sys}/{rel}'
+    USER_AGENT = ('pypureclient/1.25.0/FA/2.0/{sys}/{rel}'
```

Comment:
I thought when we release for one BU, we don't touch other BU's files. But I don't understand why :) 

Per https://wiki.purestorage.com/display/psw/Python+REST+client+for+Pure1%2C+FA%2C+FB
`For example, if you are updating a new FA SDK, and notice that some of the Pure1 files have changed, do not add those changes!`

Reply:
Yea, there were a few description and validation changes from FA that I omitted. I figured that we'd want to make sure the `USER_AGENT` for each client is correct, though. This is more a reflection of the SDK's version rather than the content of any BU's API.

Might be better/cleaner if in the future this was centralized in some way rather than stamped out across each client file.

Reply:
Works for me